DataFormats/HGCDigi/interface/HGCSample.h

```diff
@@ -64,6 +64,7 @@ class HGCSample {
    */
   void setWord(uint32_t word, uint32_t mask, uint32_t pos)
   {
+    if( word > mask ) word = mask; // deal with saturation
```

Comment:
@lgray : is this the only place the saturation bug fix is, isn't it?
are you sure this method setWord is used today? I could not find any caller...